sfcli/SmartFace.Cli/Commands/SubCamera/AddCameraCmd.cs

```diff
@@ -0,0 +1,51 @@
+using System.ComponentModel.DataAnnotations;
+using System.Threading.Tasks;
+using McMaster.Extensions.CommandLineUtils;
+using Newtonsoft.Json;
+using SmartFace.Cli.Common;
+using SmartFace.Cli.Core.ApiAbstraction;
+using SmartFace.Cli.Core.ApiAbstraction.Models;
+
+namespace SmartFace.Cli.Commands.SubCamera
+{
+    [Command(Name = "add", Description = "Create new camera")]
+    public class AddCameraCmd : BaseCameraModifyingCmd
+    {
+        [Required]
```

Comment:
is not [Required] attribute redundant?

Reply:
Yes, the attribute with the check in `if (!Name.HasValue)` is redundant. But I personally would like to keep the attribute, as that enabled the validation to take place in the cli library before our code is called, and potentially display better information to the user then we can, as the library has all the context.

So if one of them is to be removed, I would prefer to remove the `if (!Name.HasValue)` check. What do you think?

Reply:
i would also prefer removing if condition

Reply:
done